examples/api/test/widgets/shortcuts/logical_key_set.0_test.dart

```diff
@@ -0,0 +1,118 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+import 'package:flutter_api_samples/widgets/shortcuts/logical_key_set.0.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  Future<void> sendKeyCombination(
+    WidgetTester tester,
+    List<LogicalKeyboardKey> keys,
+  ) async {
+    for (final LogicalKeyboardKey key in keys) {
+      await tester.sendKeyDownEvent(key);
+    }
+    for (final LogicalKeyboardKey key in keys.reversed) {
+      await tester.sendKeyUpEvent(key);
+    }
+  }
+
+  testWidgets('Verify correct labels are displayed', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const example.LogicalKeySetExampleApp(),
+    );
+
+    expect(find.text('LogicalKeySet Sample'), findsOneWidget);
+    expect(
+      find.text('Add to the counter by pressing Ctrl+C'),
+      findsOneWidget,
+    );
+    expect(find.text('count: 0'), findsOneWidget);
+  });
+
+  testWidgets('LCtrl-C or C-LCtrl combination press updates counter', (WidgetTester tester) async {
```

Comment:
Consider changing key combination descriptions to make them more obvious, maybe 'LCtrl-C' -> 'CtrlLeft+C'
Also consider changing, "combination press" to "key combination".
(Same for the tests below)

Reply:
Thanks for the suggestions!